app/services/workflow_executions/cancel_service.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module WorkflowExecutions
+  # Service used to initiate the cancel of a WorkflowExecution
```

Comment:
Change `cancel` to `cancelation` in this comment

Reply:
Fixed, thanks